test/e2e/kubelet.go

```diff
@@ -395,15 +441,19 @@ var _ = framework.KubeDescribe("kubelet", func() {
 			testTbl := []hostCleanupTest{
 				{
 					itDescr: "after deleting the nfs-server, the host should be cleaned-up when deleting sleeping pod which mounts an NFS vol",
```

Comment:
Very small nit - this could be clearer by saying something like
>"after stopping the nfs server pod, deleting the client pod should remove it's pod directory from the node.